### PR TITLE
Integration of contracts 1.0 and NFT flows

### PR DESCRIPTION
**Pull request by aaitor**
## Description

It brings contracts `v1.0` to the sdk and implement the new NFT service agreements

## Is this PR related with an open issue?

Related to Issue #43

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [X] Follows the code style of this project.
- [X] Tests Cover Changes
- [X] Documentation

### CODE REVIEW

File: src/main/java/io/keyko/nevermined/api/AgreementsAPI.java [hunk 1]
```diff
@@ -53,4 +41,13 @@
      * @throws ServiceAgreementException Exception
      */
     boolean isAccessGranted(String agreementId, DID did, String consumerAddress) throws ServiceAgreementException;
+
+    /**
+     * Get the status of a service agreement.
+     *
+     * @param agreementId id of the agreement
+     * @return AgreementStatus with condition status of each of the agreement's conditions.
```
Comment (r-marques) [inline src/main/java/io/keyko/nevermined/api/AgreementsAPI.java:49]: This now returns the `Agreement` and not only the status

File: src/main/java/io/keyko/nevermined/api/config/NeverminedConfig.java [hunk 2]
```diff
@@ -272,11 +315,11 @@ public NeverminedConfig setDidRegistryAddress(String address) {
     }
 
     public String getEscrowPaymentConditionConditionsAddress() {
-        return escrowRewardAddress;
+        return escrowPaymentAddress;
     }
 
     public NeverminedConfig setEscrowPaymentConditionConditionsAddress(String address) {
-        this.escrowRewardAddress = address;
+        this.escrowPaymentAddress = address;
```
Comment (r-marques) [inline src/main/java/io/keyko/nevermined/api/config/NeverminedConfig.java:322]: Is the double `Condition` in the name a typo?

File: src/main/resources/sla/sla-did-sales-conditions-template.json [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      },
+      {
+        "name": "_amounts",
+        "type": "uint256[]",
+        "value": {parameter.amounts}
+      },
+      {
+        "name": "_receivers",
+        "type": "address[]",
+        "value": {parameter.receivers}
```
Comment (r-marques) [inline src/main/resources/sla/sla-did-sales-conditions-template.json:104]: should the value be in quotes?

**Review by r-marques — APPROVED** [review]
lgtm

File: src/main/java/io/keyko/nevermined/api/config/NeverminedConfig.java (hunk 2 above)
Comment (aaitor) [reply src/main/java/io/keyko/nevermined/api/config/NeverminedConfig.java:322]: yes, it is, fixing it

File: src/main/java/io/keyko/nevermined/api/AgreementsAPI.java (hunk 1 above)
Comment (aaitor) [reply src/main/java/io/keyko/nevermined/api/AgreementsAPI.java:49]: adding more details to the documentation

File: src/main/resources/sla/sla-did-sales-conditions-template.json (hunk 3 above)
Comment (aaitor) [reply src/main/resources/sla/sla-did-sales-conditions-template.json:104]: this file is just a template, the output generated include in this case an array of strings

**Pull request merged**